scripts/vktrace_file_generator.py

```diff
@@ -3330,7 +3330,7 @@ def GenerateTraceVkPacketsHeader(self):
                             or (p.name in ['pFeatures'] and 'nvx' in p.type.lower())
                             or (p.name in ['pFeatures', 'pProperties','pFormatProperties','pImageFormatInfo','pImageFormatProperties','pQueueFamilyProperties',
                                            'pMemoryProperties','pFormatInfo','pSurfaceFormats','pMemoryRequirements','pInfo',
-                                           'pSparseMemoryRequirements','pSurfaceCapabilities'] and '2khr' in p.type.lower())
+                                           'pSparseMemoryRequirements','pSurfaceCapabilities'] and ('2khr' in p.type.lower() or '2' in p.type.lower()))
```

Comment:
Is the check " '2khr' in p.type.lower()" redundant now that '2' in p.type.lower() is in the condition?

Reply:
'2' in p.type.lower() is the change added in this commit

Reply:
yes, it is redundant, will remove it